write_to_file.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Write to file.")
+    parser.add_argument("contents", help="Contents.")
+
+    args = parser.parse_args()
+
+    f = open("./build/config.h", "a")
```

Comment:
This (build directory) needs to be passed to the script as an argument

Reply:
Done!